modules/nested-tabs/assets/js/base-nested-tabs.js

```diff
@@ -61,13 +37,16 @@ export default class BaseNestedTabs extends Base {
 		};
 	}
 
-	activateDefaultTab() {
+	activateDefaultTab( openDefaultTab = true ) {
```

Comment:
Remove this param, in mega menu check for edit mode before calling it.

Reply:
I have a problem here, the `activateDefaultTab` runs in the parent class `onInit()` therefore bounding the function in a condition won't do any difference. 
I'd like to hear your suggestion regarding this one.

Reply:
Done